src/main/java/setavailability/SetAvailabilityController.java

```diff
@@ -0,0 +1,23 @@
+package setavailability;
+
+/**
+ * Class responsible for initializing the first-time setting of availability;
+ * takes input of set of LocalTime values & gives it to SetAvailability interactor.
+ *
+ * @author: od-obas1187
+ */
+public class SetAvailabilityController {
+    public static SetAvailabilityInputBoundary userAvailabilityInput;
+
+    /**
+     * 1. Receives set of String values (user’s availabilities);
+     * 2. Sets the availabilities inputted by the user, sends it to the
+     * SetAvailabilityInputBoundary
+     *
+     * @param availabilityInputs
+     */
+    public SetAvailabilityController(String[] availabilityInputs) {
```

Comment:
Similarly to my last comment, this controller does not give the driver a way to provide it with a particular interaction class upon which to depend. In Prof. Gries lectures, as well as in David's use case code, you will find that Controllers are passed in Interactor objects (disguised as InputBoundaries) as construction parameters, and then have a separate method that actually calls it. I would restructure this to have a constructor and a separate, functional method that the UI can call from.